examples/src/main/java/com/cloudera/nav/plugin/client/examples/updatedResults/GetAllUpdatedResults.java

```diff
@@ -0,0 +1,32 @@
+package com.cloudera.nav.plugin.client.examples.updatedResults;
+
+import com.cloudera.nav.plugin.client.*;
+
+import java.util.Map;
+
+/**Example for using getAllUpdated() methods to retrieve entities and relations modified since a given set of extractions.
```

Comment:
Make sure you stick to 80-characters per line. In IntelliJ you can set this up so there's a marker to guide you